Tools/build/stable_abi.py

```diff
@@ -7,22 +7,22 @@
 (relative to the manifest file, as they appear in the CPython codebase).
 """
 
-from functools import partial
-from pathlib import Path
-import dataclasses
-import subprocess
-import sysconfig
 import argparse
-import textwrap
-import tomllib
+import csv
+import dataclasses
 import difflib
-import pprint
-import sys
+import io
 import os
 import os.path
-import io
+import pprint
 import re
-import csv
```

Comment:
A shame to loose the fun length-wise sorting here!

Reply:
Yes.
It's just an insignificant personal feeling, but for me, this does suck out a lot of the the fun of contributing to the project.